fdbserver/MutationTracking.cpp

```diff
@@ -106,9 +106,7 @@ TraceEvent debugTagsAndMessageEnabled(const char* context, Version version, Stri
 			MutationRef m;
 			BinaryReader br(mutationData, AssumeVersion(rdr.protocolVersion()));
 			br >> m;
-			if (m.isEncrypted()) {
-				throw encrypt_unsupported();
-			}
+			ASSERT(!m.isEncrypted());
```

Comment:
Assert also throws, just remove this.

Reply:
Basically nothing bad happens if the mutation is encrypted, the debugMutation() message generator just won't be able to do much with it but it won't fail.

Reply:
ok. I did it to make sure in simulation that my understand of the code is correct, that indeed no encrypted mutation is passed into the mutation tracker. but I can remove it.

Reply:
Did you enable the mutation tracker in your runs?  It's just a debugging tool and normally disabled.  It's possible that the only places that call into it are working with plaintext mutations, however even if that is the case I think this assert is not needed because there could be a place that deals with encrypted or plaintext mutations that calls into the tracker regardless.

Reply:
Make sense. And I didn't aware it needs to be enabled.